src/main/java/org/opensearch/knn/jni/JNIService.java

```diff
@@ -253,4 +253,27 @@ public static long transferVectors(long vectorsPointer, float[][] trainingData)
     public static void freeVectors(long vectorsPointer) {
         FaissService.freeVectors(vectorsPointer);
     }
+
+    /**
+     * Range search index for a given query vector
+     *
+     * @param indexPointer pointer to index in memory
+     * @param queryVector vector to be used for query
+     * @param radius search within radius threshold
+     * @param engineName name of the engine to use
+     * @param indexMaxResultWindow maximum number of results to return
+     * @return KNNQueryResult array of neighbors within radius
+     */
+    public static KNNQueryResult[] radiusQueryIndex(
+        long indexPointer,
+        float[] queryVector,
+        float radius,
+        String engineName,
```

Comment:
Use Engine - we switched to this in 2.13

Reply:
ack